crates/miden-tx/src/host/mod.rs

```diff
@@ -889,17 +904,17 @@ where
     /// Checks if the necessary witness for accessing the asset is already in the merkle store,
     /// and if not, extracts all necessary data for requesting it.
     ///
-    /// Expected stack state: `[ASSET, vault_root_ptr]`
+    /// Expected stack state: `[event, ASSET, vault_root_ptr]`
```

Comment:
Should we update all `Expected stack state` comments here and in the exec host to include the event ID? That would probably be clearest.

Reply:
Added it for all eventhandlers.